StortSpelProjekt/EngineTests/src/test.cpp

```diff
@@ -362,15 +362,15 @@ TEST(MEMORYMANAGER, HEAPLARGE)
 }
 
 TEST(MEMORYMANAGER, HEAPFEWANDLARGE)
-{
-	void* smallMem = MemoryManager::AllocHeapBlock();
-	void* smallMemSec = MemoryManager::AllocHeapBlock();
-
-	EXPECT_NE(smallMem, nullptr);
-	EXPECT_NE(smallMemSec, nullptr);
-	
-	if(smallMem)
-		MemoryManager::FreeHeapBlock(smallMem);
+{
+	void* smallMem = MemoryManager::AllocHeapBlock();
+	void* smallMemSec = MemoryManager::AllocHeapBlock();
+
+	EXPECT_NE(smallMem, nullptr);
+	EXPECT_NE(smallMemSec, nullptr);
+	
+	if(smallMem)
```

Comment:
if-statements blocks are mandatory according to code standard. Required change here.